### PR TITLE
Column design

### PR DESCRIPTION
**Pull request by Geor23**

### CODE REVIEW

File: client/templates/clientTweetWall.html [hunk 1]
```diff
@@ -1,8 +1,21 @@
...
-    src="'/templates/tweetColumn.html'" 
-    ng-repeat="displayColumn in displayColumns track by $index" 
-    ng-init="colIndex = $index">
-    </ng-include>
-</md-cotent>
+<md-content layout="column" class="container" ng-style="{'overflow-y' : (loggedIn ? 'auto' : 'hidden')}">
+	<div ng-if="adminView" class="columns-header">
+		<div class="column-indicator">
+			<p class="column-indicator-writing">Pinned</p>
+		</div>
```
Comment (stephen-lt-sl) [inline client/templates/clientTweetWall.html]: This will have my approval if you get rid of the double sized indentation here :rage:

**Review by stephen-lt-sl — COMMENTED** [review]
Besides the comment, looks good - although imo you could probably make it a little smaller, it takes up just a bit more vertical space than I think is ideal

**Comment by susanthomson** [comment]
I think this looks good but would look better if we got rid of the border on the admin-container-inside so the horizontal lines aren't competing with each other.

**Review by susanthomson — APPROVED** [review]
:liechtenstein:

**Pull request merged**